perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -758,6 +759,45 @@ public RichAttributeRowMapper(AttributeRowMapper attributeRowMapper, RowMapper<P
 		}
 	}
 
+	@Override
+	public List<Attribute> getAttributes(PerunSession sess, Member member, Group group) throws InternalErrorException {
+		try {
+			//member-group attributes, member core attributes
+			return jdbc.query("select " + getAttributeMappingSelectQuery("mem") + " from attr_names " +
```

Comment:
In getAttributeMappingSelectQuery i would prefer something like mem_gr to see that table is member_group. Also update this name in rest of SQL query.